explorer/src/entity/ChainlinkNode.ts

```diff
@@ -175,40 +175,30 @@ export async function uptime(db: Connection, node: ChainlinkNode | number) {
 
 // uptime from completed sessions
 async function historicUptime(db: Connection, id: number): Promise<number> {
-  const { seconds } = await db
+  const { seconds } = (await db
```

Comment:
Could we store this query in a temp variable? Wrapping awaits in parentheses for order of execution reduce readability a lot.